### PR TITLE
Add Samsung Tv Plus as a blockable service

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
https://github.com/AdguardTeam/HostlistsRegistry/issues/317

Maybe we need to block other domains as well, I don't have an android TV so I can't check.

### CODE REVIEW

**Review by Alex-302 — APPROVED** [review]

File: services/samsung_tv_plus.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+id: samsung_tv_plus
+name: Samsung TV Plus
+rules:
+  - '||samsungtvplus.com^'
+  - '||samsung.wurl.tv^'
+icon_svg: <svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="121 -91 672 672"><path d="m710.504 89.711.004 38.892c0 .073.021.14.021.214V352.97c0 21.351-17.05 38.526-38.326 38.835l-.05.052h-.473c-.015 0-.028.004-.042.004h-96.769v31.034a24.11 24.11 0 0 1-24.162 24.163H368.046a24.11 24.11 0 0 1-24.162-24.163v-31.034H208.832v35.86c0 34.394 27.687 62.083 62.081 62.083h457.22c34.395 0 62.085-27.69 62.085-62.083V151.795c0-34.394-27.69-62.084-62.084-62.084zM185.028 0c-34.394 0-62.084 27.69-62.084 62.084V329.78c0 34.393 27.69 62.081 62.083 62.081h23.804V128.817c0-21.546 17.346-38.89 38.892-38.89h38.6l.176-.216h424.005V62.084C710.504 27.69 682.814 0 648.421 0Z"/></svg>
```
Comment (slavaleleka) [inline services/samsung_tv_plus.yml:6]: logo is a bit different
https://uploads.adguard.org/sllftdcgl0alx.png

https://www.samsung.com/us/televisions-home-theater/tvs/tvplus/
https://logos.fandom.com/wiki/Samsung_TV_Plus#2022%E2%80%93present

Comment (Sergey-Lyapin) [reply services/samsung_tv_plus.yml:6]: The letters will be very hard to read, in my opinion they don't make sense.

Comment (slavaleleka) [reply services/samsung_tv_plus.yml:8]: ok. and what about colors?

Comment (Sergey-Lyapin) [reply services/samsung_tv_plus.yml:8]: According to the requirements the icon must be black and white:

```
SVG icon requirements:

The icon must have a square shape, i.e. the third and fourth parameters of the viewBox attribute must be equal.
The icon must be black and white.
It should be as small as possible.
Svg tag must contain fill="currentColor" attribute. This is important for themes to work properly.
Svg tag must not contain width and height attributes
Double quotes inside the SVG tag must be escaped so as not to break JSON.
```

**Review by slavaleleka — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

File: services/samsung_tv_plus.yml (hunk 1 above)
Comment (Alex-302) [reply services/samsung_tv_plus.yml:8]: How icons look
image

In any case it is hard to find the necessary service only by icon. Text search is required.

**Pull request merged**